### DISCUSSION TITLE
Disable redundant python venv prompt on powershell

### DISCUSSION THREAD

**Discussion by karlosos** [starship/starship#2087] [Q&A] [answered]
Is it possible to disable (1) and still having starships `venv` indicator (2)?

![image](https://user-images.githubusercontent.com/3882385/103773186-f5f11c80-502a-11eb-8248-8c88ffde48d8.png)

**Comment by davidkna** [comment, marked as answer]
Can you try running `$ENV:VIRTUAL_ENV_DISABLE_PROMPT=1` before activating the `venv`?

**Reply by karlosos** [reply]
Thanks. It's working. 

![image](https://user-images.githubusercontent.com/3882385/103774111-5af94200-502c-11eb-942f-c5464965ddd4.png)

After this fix I had a problem with empty line, but I saw it was discussed before. https://github.com/starship/starship/discussions/1609

## Solution

Add `$ENV:VIRTUAL_ENV_DISABLE_PROMPT=1` to `$PROFILE` with `vim $PROFILE`.

```
# Import-Module posh-git
Invoke-Expression (&starship init powershell)

# Disable env prompt
$ENV:VIRTUAL_ENV_DISABLE_PROMPT=1
```

Remove empty line by adding `add_newline = false` to your `~\.config\starship.toml` file.

Now everything works great. Thank you, @davidkna.

![image](https://user-images.githubusercontent.com/3882385/103774750-5b460d00-502d-11eb-9c63-5728d52729c1.png)
